vitest.config.mjs

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
+
+export default defineConfig({
```

Comment:
Let's dig into options more. Can we get vitest to output coverage reports?

Reply:
Yes, I can! However, coverage isn't useful until the majority of tests are running in vitest. It's configured, but disabled for now.